documentation/api/io.strimzi.api.kafka.model.AclRule.adoc

```diff
@@ -0,0 +1,93 @@
+Use `ACLRule` properties to configure simple authorization for a `KafkaUser`.
```

Comment:
The type name is `AclRule`. But it would be better to avoid using the type name at all because from the point of view of someone wanting the write valid YAML the type name is irrelevant, it's the structure that counts. This point seems to apply to all the description files in this PR.

For this specific example:

```suggestion
Configures a access control rule for a `KafkaUser` when brokers are using the `AclAuthorizer`.
```

Reply:
> Configures access control rule for a `KafkaUser` when brokers are using the `AclAuthorizer`.